azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -72,13 +68,14 @@ public ResourceRetriever getJWTResourceRetriever() {
     }
 
     @PostConstruct
-    private void trackCustomEvent() {
+    private void sendTelemetry() {
         if (aadAuthProps.isAllowTelemetry()) {
             final Map<String, String> events = new HashMap<>();
+            final TelemetrySender sender = new TelemetrySender();
```

Comment:
Curious whether we need to create a new instance every time.

Reply:
I decide to create new instance here with following reasons:

* azure-spring-boot has no option for disbale all telemetry, need to keep singleton bean for sender.
* sender is tiny with minor cost when creation.
* according to singleton bean, this way is thread safe.
* the method will be called only once when starter boot up.